packages/main/src/resolvers/conversation.ts

```diff
@@ -91,6 +97,30 @@ export const ConversationMutations: ConversationMutationsResolvers = {
     return thread
   },
 
+  async flag(_parent, { ids, isFlagged }) {
+    let threads: C.Conversation[] = []
+    ids.forEach(id => {
```

Comment:
The more idiomatic way to iterate over an array is `for (const id of ids) { ... }`